### PR TITLE
Hot loading

### PR DESCRIPTION
**Pull request by MoOx**
Last commit is stupid since we don't use webpack-dev-server. But we still need a client to handle the HMR messages?

TODO after basic HMR ok via livereload: add react-hot-loader@next (3 beta) (but when HMR is ok, should not be a problem)

### CODE REVIEW

**Comment by MoOx** [comment]
I was tired. I just need to replace client with webpack-hot-middleware.

**Comment by MoOx** [comment]
Ready for review.

File: packages/phenomic-plugin-renderer-react/src/render/client.js [hunk 1]
```diff
@@ -18,11 +20,13 @@ function render(routes: () => React$Element<any>) {
   }
 
   const initialStateNode = document.getElementById("Hydration")
-  const store = createStore(
-    initialStateNode && initialStateNode.textContent
-      ? JSON.parse(initialStateNode.textContent)
-      : undefined,
-  )
+  store =
+    store ||
```
Comment (MoOx) [inline packages/phenomic-plugin-renderer-react/src/render/client.js]: This change is required for hot loading to work properly (as we re-render the full app). @bloodyowl can you confirm this won't cause any trouble for static rendering? (as it's in render/client.js ^^),

Comment (bloodyowl) [inline packages/phenomic-plugin-renderer-react/src/render/client.js]: not that beautiful but it should do without effects on SSR

**Pull request merged**